### PR TITLE
Civi-test-run add in function to run phpunit tests within the drupal …

### PR DESCRIPTION
**Pull request by seamuslee001**
…folder

ping @totten the only issue for me is that should i use phpunit4 or phpunit for this. The later is probably consistent with 4.6 but not 100% sure

### CODE REVIEW

File: bin/civi-test-run [hunk 1]
```diff
@@ -179,6 +189,7 @@ for TESTTYPE in $SUITES ; do
     phpunit-crm)  task_phpunit CRM_AllTests ;;
     phpunit-api)  task_phpunit api_v3_AllTests ;;
     phpunit-civi)  task_phpunit 'Civi\AllTests' ;;
+    phpunit-drupal) phpunit_drupal ;;
```
Comment (totten) [inline bin/civi-test-run]: @seamuslee001 Nice! I'd really want to get the `civicrm-drupal` suite running.

Minor nitpick: the other functions have a naming convnetion (`task_karma`, `task_upgrade`, `task_phpunit`). This one should be `task_phpunit_drupal`.

Comment (seamuslee001) [reply bin/civi-test-run]: changed in the latest commit @totten

**Comment by seamuslee001** [comment]
@totten I think this is good to merge now Tim

**Comment by totten** [comment]
Cool :)

**Pull request merged**